packages/rocketchat-ui/client/components/header/header.js

```diff
@@ -104,25 +104,38 @@ Template.header.helpers({
 });
 
 Template.header.events({
-	'click .iframe-toolbar button'() {
+	'click .iframe-toolbar .js-iframe-action'(e) {
 		fireGlobalEvent('click-toolbar-button', { id: this.id });
+		$('button', e.currentTarget).blur();
+		return false;
 	},
 
 	'click .rc-header__toggle-favorite'(event) {
 		event.stopPropagation();
 		event.preventDefault();
-		return Meteor.call('toggleFavorite', this._id, !$(event.currentTarget).hasClass('favorite-room'), function(err) {
-			if (err) {
-				return handleError(err);
+		return Meteor.call(
+			'toggleFavorite',
+			this._id,
+			!$(event.currentTarget).hasClass('favorite-room'),
+			function(err) {
+				if (err) {
+					return handleError(err);
+				}
```

Comment:
Could you avoid some extra lines using an arrow function?

Reply:
yes we can!